example/example/tests/test_customer_views.py

```diff
@@ -55,6 +55,7 @@ def test_create_customer(self, user, mock_storage):
 
         customer_data = factory.build(dict, FACTORY_CLASS=CustomerFactory)
         customer_data["type"] = customer_data.pop("customer_type")
+        customer_data["comp_type"] = customer_data.pop("company_type")
```

Comment:
Again, why use different names?  (If you use the same name, you won't need to have this line here)

Reply:
I will look into this